api/README.md

```diff
@@ -16,11 +16,26 @@ Additionally, you need to specify [EMULATED](ENVIRONMENT.md#emulated) flag equal
 
 #### Steps
 To run the CodePush Server locally, follow these steps:
-1. Clone the CodePush Service repository.
-1. Create a `.env` file and configure the mandatory variables as outlined in the `ENVIRONMENT.md` file.
-1. Install dependencies by running `npm install`.
-1. Build the server by running `npm run build`.
-1. Start the server by running `npm run start:env`.
+
+1. Clone the CodePush Service repository to your local machine.
+
+2. Copy the `.env.example` file to a new file named `.env` in the root directory:
+   ````bash
+   cp .env.example .env
+   ````
+   Fill in the values for each environment variable in the `.env` file according to your development or production setup.
```

Comment:
Only mandatory variables are needed - please mark mandatory values in `.env.example` file.

Reply:
It’s best to list all environment variables that the app uses, even if some are optional. This ensures transparency and clarity for developers who may work on or deploy the application in different environments.

How exactly should be "marked" the mandatory values ?

Reply:
Up to you - e.g.

```
# Required, the URL of your server
SERVER_URL=http://localhost:3000
```